src/Service/ServiceExe.cs

```diff
@@ -1489,26 +1489,6 @@ public FOService(ILogger<FOService> logger)
 
                 ServiceLogger = logger;
```

Comment:
Can you add a comment here saying something similar to: "Only initialize the bare minimum required fields & properties here. Most initialization should be done in ExecuteServerBootTasks() as part of Service Start()."